tests/__init__.py

```diff
@@ -127,29 +127,30 @@ def test_model_validation(self):
             'datetime_local':'2012-08-13T05:25:04.362-03:30'
         }))
         response_error(resp)
-        errors = json.loads(resp.data)
+        errors = json.loads(resp.data.decode('utf-8'))
+
         self.assertTrue('field-errors' in errors)
-        self.assertEqual(errors['field-errors'].keys(), ['email'])
+        self.assertEqual(list(errors['field-errors'].keys()), ['email'])
 
         resp = self.app.put('/user/%s/' % self.user_1_obj['id'], data=json.dumps({
             'email': 'invalid',
             'first_name': 'joe',
             'last_name': 'baker',
         }))
         response_error(resp)
-        errors = json.loads(resp.data)
+        errors = json.loads(resp.data.decode('utf-8'))
         self.assertTrue('field-errors' in errors)
-        self.assertEqual(errors['field-errors'].keys(), ['email'])
+        self.assertEqual(list(errors['field-errors'].keys()), ['email'])
 
         resp = self.app.put('/user/%s/' % self.user_1_obj['id'], data=json.dumps({
             'emails': ['one@example.com', 'invalid', 'second@example.com', 'invalid2'],
         }))
 
         response_error(resp)
-        errors = json.loads(resp.data)
+        errors = json.loads(resp.data.decode('utf-8'))
         self.assertTrue('field-errors' in errors)
-        self.assertEqual(errors['field-errors'].keys(), ['emails'])
-        self.assertEqual(errors['field-errors']['emails']['errors'].keys(), ['1', '3'])
+        self.assertEqual(list(errors['field-errors'].keys()), ['emails'])
+        self.assertCountEqual(list(errors['field-errors']['emails']['errors'].keys()), ['3', '1'])
```

Comment:
Good catch, keep in mind that `assertCountEqual` was only added in Python 3.2 and therefore would break Python 2.7 compatibility. I'll fix this by simply doing a `set()` comparison.